### ISSUE TITLE
Accessibility: Screen Reader Support

### ISSUE THREAD

**Issue by chigkim** [earendil-works/pi#4687]
### What happened?

TUI utilizes bunch of ascii arts characters in borders, dynamic decorations, selector formatting, footer layout, etc, and it is noisy and confusing when read aloud by screen readers.
For example, screen readers keeps repeating "box drawing light horizontal" bunch of times before and after prompt field.
Also it uses braille unicode for spinners. The characters are supposed to be used for Braille display.
It is hard to find the start of assistant response without visual marking.

### Steps to reproduce

Run pi in terminal, and read the UI with VoiceOver on Mac.

### Expected behavior

The cli should have -sr flag and/or `"screenreader": true` in settings.json to make TUI easier to use with screen readers by enabling a flatter, less decorative rendering mode.
Keep the existing visual TUI behavior unchanged when the setting is not active.
Also add User: and Assistant: to mark the start of message role.
Gemini-cli has this feature.
I have a vibe coded branch as a concept.
https://github.com/chigkim/pi/tree/screenreader
Thanks!

### Version

v0.75.3

**Comment by badlogic** [comment]
i would love to fix this, but i currently lack the resources. this would require a complete rewrite of the entire TUI.

**Comment by chigkim** [comment]
Could you check out my forked branch? It's not as daunting as it sounds.
https://github.com/chigkim/pi/tree/screenreader
You just need to implement -sr flag or screenReader in settings.json, and when -sr is specified:
1. Remove ascii arts and if necessary replace with appropriate ones like > for selection, I for input, O for output, etc.
2. Remove Leading column pad, so it doesn't say "space" unnecessarily every line, (keep the spaces for code block indentation though)
3. Label with User:, Tool:, Assistant:, etc.
The regular UI stays the same.
Thanks!

**Comment by chigkim** [comment]
@badlogic, In case you get around it, I renamed my forked branch to screenreader from add-sr-flag.
https://github.com/chigkim/pi/tree/screenreader
Hope you consider! Thanks!

**Comment by Oliver2213** [comment]
I would also find this very helpful. Claude code also has this problem so it would be nice to see one of these with it fixed.

**Comment by vick08** [comment]
As a screen reader user myself, I would like to echo and promote this request. I really love the Pi agent, but with the current state of TUI using the product is quite tedious for the reasons listed in the original request.
If you feel that the provided branch does not meet the needs and the quality standards of the project, please let us know, and we would be happy to put our heads together to facilitate the fixes.
Myself and the rest of the community would really appreciate your attention to making Pi accessible and usable for screen reader users.
